typescript/create-onchain-agent/templates/mcp/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "ES2022",
+    "module": "Node16",
+    "moduleResolution": "Node16",
+    "outDir": "./build",
+    "rootDir": "./src",
+    "strict": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true
```

Comment:
we should add the decorator compiler configs so that people can extend with new action providers without compilation errors related to the reflection metadata tagging
@0xRAG